$NetBSD: patch-de,v 1.1 2007/05/28 08:13:26 dan Exp $
#
# old_revision [f92dd754bf5c1e6eddc9c462b8d68691cfeb7f8b]
#
# patch "work.cc"
#  from [0b6a0b54ff0eeab3092ad1f8fd73bb2af2308847]
#    to [9b78a7cb064436d0eed809eaa14c46b548b46096]
#
============================================================
--- work.cc	0b6a0b54ff0eeab3092ad1f8fd73bb2af2308847
+++ work.cc	9b78a7cb064436d0eed809eaa14c46b548b46096
@@ -732,7 +732,8 @@ struct content_merge_empty_adaptor : pub
   virtual void get_version(file_id const &, file_data &) const
   { I(false); }
   virtual void record_merge(file_id const &, file_id const &,
-                            file_id const &, file_data const &,
+                            file_id const &, 
+                            file_data const &, file_data const &,
                             file_data const &)
   { I(false); }
   virtual void get_ancestral_roster(node_id, boost::shared_ptr<roster_t const> &)
